docs/rst/tutorials/core/deployment/ds_locators_runtime/ds_locators_runtime.rst

```diff
@@ -0,0 +1,369 @@
+            Server Addresses:   UDPv4:[0.0.0.0]:11811
+
+    .. tab:: Server 1
+
+        .. code-block:: bash
+
+            ### Server is running ###
+            Participant Type:   SERVER
+            Server ID:          1
+            Server GUID prefix: 44.53.01.5f.45.50.52.4f.53.49.4d.41
+            Server Addresses:   UDPv4:[0.0.0.0]:11811
+
+The output displays the ``server id`` set, followed by the server GUID prefix.
+Server addresses ``0.0.0.0`` net mask allows all addresses to be set, but we need each server specific address.
+Finally, the ``11811`` default port would be necessary for further configuration.
+
+In order to set-up the *clients* to discover each other through the *server*, we need to set the *server* data (IP
+address, port and prefix) in all *clients*.
+The port and prefix are already known, and each specific *server* IP address can be obtained introducing the container
+names in the following command:
+
+.. tabs::
+
+    .. tab:: Discovery Environment 0
+
+        .. code-block:: bash
+
+            docker inspect --format '{{ .NetworkSettings.IPAddress }}' server_0
+
+    .. tab:: Discovery Environment 1
+
+        .. code-block:: bash
+
+            docker inspect --format '{{ .NetworkSettings.IPAddress }}' server_1
+
+.. note::
+
+    For this example, these are the used IP addresses for each entity.
+
+    +----------------+------------+
+    | CONTAINER NAME | IP ADDRESS |
+    +================+============+
+    | client_sub     | 172.17.0.2 |
+    +----------------+------------+
+    | client_pub_0   | 172.17.0.3 |
+    +----------------+------------+
+    | client_pub_1   | 172.17.0.4 |
+    +----------------+------------+
+    | server_0       | 172.17.0.5 |
+    +----------------+------------+
+    | server_1       | 172.17.0.6 |
+    +----------------+------------+
+
+Create a XML configuration file in each *client* container, and complete it with the following XML profile, including
+the *server* data obtained (IP address, port and GUID prefix).
+
+.. literalinclude:: /resources/tutorials/core/deployment/ds_locators_runtime/client_configuration.xml
+    :language: xml
+
+The entities need to be configured to use the created profile.
+To do so, make sure the document name is ``DEFAULT_FASTRTPS_PROFILES.xml``, and set the following environment variable in all *clients*:
+
+.. code-block:: bash
+
+    export RMW_FASTRTPS_USE_QOS_FROM_XML=1
+
+``FASTDDS_ENVIRONMENT_FILE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Finally, each *client* needs to start a discovery client.
+To do so, and this is the key of the tutorial, the environment variable ``FASTDDS_ENVIRONMENT_FILE`` must be set.
+
+Setting this environment variable to an existing ``json`` file allows to load the environment variables from a file
+instead of from the environment.
+This allows to change the value of some environment variables at run time with just modifying and saving the changes
+to the file.
```

Comment:
Lots of `allow` here. Also, I'm missing a link to the Fast DDS docs about this awesome feature.